.github/workflows/plugin_install.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest]
-        jdk: [8, 11, 14]
```

Comment:
This shouldn't be updated

Reply:
Yeah my bad, I just copy+pasted and did not catch that thank you!